TranslatorReasonerAPI.yaml

```diff
@@ -928,12 +937,14 @@ components:
           description: >-
             Source of an attribute provided by this TRAPI web service.
           example: infores:chembl
+          nullable: true
         original_attribute_names:
           type: array
           items:
             type: string
           description: >-
             Names of an the attribute as provided by the source.
+          nullable: true
```

Comment:
```suggestion
          default: []
```

Reply:
Per WG discussion, we prefer null to empty array here. Instead, let's add `minItems: 1`.